app/src/main/java/com/hover/stax/bounties/SectionedBounty.java

```diff
@@ -0,0 +1,40 @@
+package com.hover.stax.bounties;
+
+import com.hover.stax.channels.Channel;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+class SectionedBounty {
+	public String header;
+	public List<Bounty> channelBounties;
+
+	public SectionedBounty(String header, List<Bounty> channelBounties) {
+		this.header = header;
+		this.channelBounties = channelBounties;
+	}
+
+	//FIXME: THIS SEARCH SHOULD NOT OCCUR IN THE UI, INSTEAD THE END OBJECT THAT BOUNTY_VIEW_MODEL SENDS.
```

Comment:
Setting the bounty adapter from two view model observers forces me to make the search in the UI, which isn't the best.